lib/Xud.ts

```diff
@@ -41,6 +57,8 @@ class Xud {
     this.logger.info('config loaded');
 
     try {
+      this.loadNodeKey(`${this.config.xudir}/nodekey.dat`);
```

Comment:
It would be better to have the util function to handle only the getting of the instance, and to decouple it from the assigning.
`this.nodeKey = this.loadNodeKey(...)`

Reply:
When I did this, I realized it really made more sense for this to be in `NodeKey.ts` after all. So I put it there for now, and we'll see if anything needs to get rearranged when we do the rpc call to encrypt/decrypt a key.